server/lib/search.js

```diff
@@ -177,10 +177,6 @@ export const searchCollectivesInDB = async (
     countryCodes = `${countries.join(',')}`;
   }
 
-  if (tags?.length) {
-    searchedTags = `{${tags.map(tag => `"${tag}"`).join(',')}}`;
-  }
-
```

Comment:
Moved this one down, it added unnecessary complexity to have two `if (tags?.length)` checks